crates/nu-data/src/base/shape.rs

```diff
@@ -31,7 +31,7 @@ pub enum InlineShape {
     ColumnPath(ColumnPath),
     Pattern(String),
     Boolean(bool),
-    Date(DateTime<Utc>),
+    Date(DateTime<FixedOffset>),
```

Comment:
Can we also do this change as a separate PR? Changing this should probably also change how dates are handled in other commands, like `date now`.

Reply:
We could keep `Date(DateTime<Utc>)` in this PR but I'm not quite sure what is the benefit? My understanding is that the `InlineShape` enum in `shape.rs` should match the `Primitive` enum, or there will be unnecessary data type conversions between `Date<DateTime<Utc>>` and `Date(DateTime<FixedOffset>)`. If I do the change in a separate PR, then I need to do these conversions in `shape.rs` in this PR.

Also, all date related subcommands including `date now` are now using the `Date(DateTime<FixedOffset>)` type in this PR, so I don't see an issue there. Maybe I missed something? What do you think?